packages/backend/src/core/DiscoveryWatcher.ts

```diff
@@ -213,12 +228,18 @@ const errorsCount = new Counter({
   help: 'Value showing amount of errors since server start',
 })
 
-function initMetrics(): (blockNumber: number, changesCounter: number) => void {
+const dailyReminderSent = new Gauge({
+  name: 'discovery_watcher_daily_reminder_sent',
+  help: 'Value showing that the daily reminder has been sent',
+})
```

Comment:
I'm not a fan of this metric. At 2AM it will tell us, that the daily reminder has been sent.